src/primitives.coffee

```diff
@@ -27,6 +27,61 @@ class ElementType
 
   set: (@content) -> @
 
+  getProperty: -> new ErrorType 'Element does not have method getProperty', @
+
+  has: -> false
+
+  map: -> new ErrorType 'Element does not have method map', @
```

Comment:
JavaScript not having a method missing really stinks. There is no way to catch any method that is missing, so for this to actually work, there has to be some register that registers all potential methods before one is initialized. Not super cool, and wondering if it's not worth it to have it.

More on this later, though.